base/src/main/resources/db/migration/V83__change_sys_tenant_to_short.sql

```diff
@@ -0,0 +1,218 @@
+            WHERE t.table_schema = 'ehr'
+              AND c.column_name = 'namespace'
+              AND t.table_type = 'BASE TABLE'
+            LOOP
+                EXECUTE format(
+                        'CREATE POLICY ehr_policy_ehrbase_migration ON ehr.%I FOR ALL TO ehrbase USING (TRUE)',
+                        table_name);
+                EXECUTE format('ALTER TABLE ehr.%I ADD COLUMN sys_tenant SMALLINT default 1', table_name);
+
+            END LOOP;
+    END
+$$;
+
+-- updates the sys_tenant column in all tables in the ehr schema with the corresponding id value from the tenant table
+-- based on the namespace column, if there is more than one row in the tenant table.
+DO
+$$
+    DECLARE
+        table_name TEXT;
+        column_name
+                   TEXT;
+    BEGIN
+        FOR table_name, column_name IN
+            SELECT c.table_name, c.column_name
+            FROM information_schema.columns c
+                     INNER JOIN information_schema.tables t
+                                ON c.table_schema = t.table_schema AND c.table_name = t.table_name
+            WHERE t.table_schema = 'ehr'
+              AND c.column_name = 'namespace'
+              AND t.table_type = 'BASE TABLE'
+            LOOP
+                IF ((SELECT count(id) FROM ehr.tenant) > 1) THEN
+                    EXECUTE format('update ehr.%I t
+            set sys_tenant = tnt.id
+            from ehr.tenant tnt
+            where tnt.tenant_id = t.namespace
+              and t.namespace != ''1f332a66-0e57-11ed-861d-0242ac120002''
+              and t.sys_tenant = 1', table_name);
+                END IF;
+            END LOOP;
+    END
+$$;
+
+-- Recreate the indexes for the tables that contain namespace
+DO
+$$
+    DECLARE
+        indexdef text;
+        index_table
+                 text;
+    BEGIN
+        FOR indexdef, index_table IN
+            SELECT pg_get_indexdef(c.oid) AS indexdef,
+                   c.relname              AS index_table
+            FROM pg_class c
+                     LEFT JOIN pg_attribute a ON a.attrelid = c.oid
+            WHERE a.attname = 'namespace'
+              and c.relname not in
+                  ('template_store_pkey', 'entry_composition_id_key', 'status_ehr_id_key', 'users_pkey',
+                   'context_participation_index')
+              and c.reltype = 0
+            LOOP
+                IF indexdef IS NOT NULL THEN
+                    EXECUTE format('DROP INDEX ehr.%I', index_table);
+                    EXECUTE (
+                        REPLACE(REPLACE(indexdef, '(namespace', ', sys_tenant'), ', namespace', ', sys_tenant'));
+                END IF;
+            END LOOP;
+    END
+$$;
+
+ALTER TABLE ehr.status
+    DROP CONSTRAINT status_ehr_id_key;
+DROP INDEX IF EXISTS status_ehr_idx;
+CREATE UNIQUE INDEX status_ehr_idx ON ehr.status (ehr_id, sys_tenant);
+ALTER TABLE ehr.status
+    ADD CONSTRAINT status_ehr_id_key UNIQUE USING INDEX status_ehr_idx;
```

Comment:
No need to create index and constraint separately
CREATE UNIQUE CONSTRAINT status_ehr_key ON ehr.status (ehr_id, sys_tenant);

Reply:
not correct syntax "CREATE UNIQUE CONSTRAINT"